### PR TITLE
Expands Surfmed among other things take 2

### PR DESCRIPTION
**Pull request by Very-Soft**
Expands Surface Medical to make it into primary medical, nerfs Space Medical, making it look a bit like things are being 'moved out' in continuing with the plan

z3
z2
z1

Also adds an ignition button to the crematorium
Also fixxes some pipe things I missed after the chapel rework
Also converts the surface cargo area into a Cargo Shop
Also adds Detective Wardrobe to detective's office
Also makes the tiny shuttle public access
Moves the reading rooms to Z2, and adds a new section to connect the area, including a new cafeteria, and a new airlock

### CODE REVIEW

**Comment by Novacat** [comment]
## UNIT_TEST: !! FAILURE !! [MAP: Area Test APC / Scrubbers / Vents Z level 1]: [1/74]Some areas lacked APCs, Air Scrubbers, or Air vents.
## UNIT_TEST: !! FAILURE !! [MAP: Active edges (all maps)]: Maps contained 2 active edges at round-start.
Active Edge /connection_edge/unsimulated (/connection_edge/unsimulated)
+--- Connecting Turf space @ 62, 28, 7
+--- Connecting Turf space @ 63, 28, 7
+--- Connecting Turf space @ 64, 28, 7
Active Edge /connection_edge/unsimulated (/connection_edge/unsimulated)
+--- Connecting Turf space @ 59, 31, 7
+--- Connecting Turf space @ 59, 32, 7

Seems like it is complaining about Virology and that there are some turfs with atmos connected to turfs without atmos on Z7 (which is Asteroid 3 I believe).

**Comment by Wickedtemp** [comment]
Love the scanner near the OR's. Overall, looks good. Only thing I'd push for is a secondary entrance from hall to medbay. On Box, it went from the hall on the left, straight to the middle of the medbay. Polaris had one coming from the southern side. Currently on Asteroid, when the reception area is crowded, we can use the EMT's entrance. I'd kinda like something like that here, if possible. We may also have a couple issues with resleeving patients now needing escorts, but those also aren't really issues, it's been like this before on previous maps so it's not really a big deal. If we can restructure the maint tunnel section just south of z2 Med's stairs to look more like a proper hall that resleeved patients can leave through, then that'd be neato. If not, we can work with it.

**Comment by Very-Soft** [comment]
> Love the scanner near the OR's. Overall, looks good. Only thing I'd push for is a secondary entrance from hall to medbay. On Box, it went from the hall on the left, straight to the middle of the medbay. Polaris had one coming from the southern side. Currently on Asteroid, when the reception area is crowded, we can use the EMT's entrance. I'd kinda like something like that here, if possible. We may also have a couple issues with resleeving patients now needing escorts, but those also aren't really issues, it's been like this before on previous maps so it's not really a big deal. If we can restructure the maint tunnel section just south of z2 Med's stairs to look more like a proper hall that resleeved patients can leave through, then that'd be neato. If not, we can work with it.

It's not in the screenshot cuz I haven't updated it yet but there is a door into the hall south of the CMO's office now.

Also, part of the point of this design is that you can turn someone loose in the main hall where they are likely to be resleeved/operated on/visiting someone in the patient rooms, and they would be able to get to the front door without getting into anything, or seeing anything especially traumatic.

**Comment by TheFurryFeline** [comment]
Note from discord, just for posterity: add the new areapath for the airlock to tether_defines.dm. lowernortheva replaces the old cargo/mining/airlock path.

```    
unit_test_exempt_from_atmos = list(
        /area/engineering/atmos_intake, // Outside,
        /area/rnd/external, //  Outside,
        /area/tether/surfacebase/mining_main/external, // Outside,
        /area/tether/surfacebase/lowernortheva, //  It's an airlock,
        /area/tether/surfacebase/emergency_storage/rnd,
        /area/tether/surfacebase/emergency_storage/atrium)
```

**Comment by Wickedtemp** [comment]
Neato!

Is Asteroid Med being downsized in this pr as well, or is that gonna be a different one?

Also before I forget, there's some nano-ui thing you'll need to update to make sure the suit-sensors map actually shows all the changes, forgot how to actually do that but iirc the steps are relatively quick and easy

**Comment by TheFurryFeline** [comment]
> Neato!
> 
> Is Asteroid Med being downsized in this pr as well, or is that gonna be a different one?
> 
> Also before I forget, there's some nano-ui thing you'll need to update to make sure the suit-sensors map actually shows all the changes, forgot how to actually do that but iirc the steps are relatively quick and easy

Changed_Medical-Asteroid_3
Changed_Medical-Asteroid_2
Looks like asteroid med indeed is being downsized but also looks to be only a partial. There's still some functionality within for some use.

**Comment by Novacat** [comment]
Honestly too much work considering the 6-7 Z levels are eventually going to be completely removed.

**Comment by Very-Soft** [comment]
> I don't see how it's too much work to blank out that old viro section and fill it in with walls, sand, and space respectively? Just seems a bit unnecessary to have all that metal foam there.

The way thing sare for spacemed right now is to imply that they are moving out. Sure I could just delete old Viro and block up all the windows, or fill in the old places with rocks to make it look normal, but I think that's less interesting from an IC perspective. And as Ascian says, being as Z6 and Z7 will be gone one day, it's a temporary thing.

**Comment by TheFurryFeline** [comment]
Fair nuff, then.

File: maps/tether/tether_defines.dm [hunk 1]
```diff
@@ -155,6 +155,12 @@
 		/area/tether/surfacebase/emergency_storage/rnd,
 		/area/tether/surfacebase/emergency_storage/atrium)
 
+//Medical Move
+	unit_test_exempt_from_atmos = list(
+		/area/tether/surfacebase/lowernortheva, // it airlock
+		/area/tether/surfacebase/lowernortheva/external //it outside
```
Comment (TheFurryFeline) [inline maps/tether/tether_defines.dm:159]: This second line is unneeded as by default, anything a child of lowernortheva will inherit the exemption, IIRC.

Comment (TheFurryFeline) [reply maps/tether/tether_defines.dm]: Alsooooooo, can, uhhh, remove my TFF comment since it no longer is relevant, just FYI.

**Comment by Very-Soft** [comment]
\o/

I will now sleep for 10000 years

**Comment by Wickedtemp** [comment]
i vote that spacemed viro is turned into a wolpin clubhouse

**Comment by TheFurryFeline** [comment]
Hue, I likey

**Comment by foopwo** [comment]
If surface medical is becoming the 'main' medical. It should probably have its own power substation instead of the one currently shared between it and security.

**Comment by KasparoVy** [comment]
Surf's up yo

**Comment by TheFurryFeline** [comment]
Oh, bugger, that definitely is a big issue, needing its own SMES...

**Pull request merged**